edison/README.md

```diff
@@ -72,3 +72,27 @@ members of the class `WordFeatureExtractorFactory`.
 | `wordCase`                          | The word, without changing the case                                     |
 | `dateMarker`                        | An indicator for whether the token is a valid date                      |
 
+##FEX Functionality
+
+In addition to the programmatic generation using feature extractors, we allow specification of
+feature extractors through a special file that defines feature extractors. This file is the .fex file.
+
+## What happens
```

Comment:
This is subsection of "FEX functionality"; right?  So maybe triple-hash-tag?: "###"